### PR TITLE
Fixes missing windoors in Delta security (And a few other places)

### PR DESCRIPTION
**Pull request by Adrer**
## What Does This PR Do
Fixes all the windoors that are missing in Deltastation's security department.
Fixes missing windoor at Deltastation's HOP's office.
Fixes: #22536

## Why It's Good For The Game
Missing windoors bad.

## Images of changes
Restores brig windoors to state before #22168 see mapdiffbot for changes.

## Changelog
:cl:
Fix: Added all the windoors that were missing in Deltastation's security department.
Fix: Missing windoor at Deltastation's HOP's office has been replaced.
/:cl:

### CODE REVIEW

**Review by warriorstar-orion — CHANGES_REQUESTED** [review]
Thanks for fixing this. I'm not sure what caused this, my only guess might be it having something to do with multiple windoors on the same tile, but UpdatePaths shouldn't care about that...

File: _maps/map_files/Delta/delta.dmm [hunk 1]
```diff
@@ -81083,7 +81108,7 @@
 "ebU" = (
 /obj/machinery/computer/general_air_control{
 	name = "Bomb Mix Monitor";
-	autolink_sensors = list("burn_sensor"="Burn Mix");
+	autolink_sensors = list("burn_sensor"="Burn  Mix");
```
[the remark this answers is not in the document]
Comment (SteelSlayer) [reply _maps/map_files/Delta/delta.dmm:81111]:
```suggestion
	autolink_sensors = list("burn_sensor"="Burn Mix");
```

**Review by Contrabang — CHANGES_REQUESTED** [review]
det's office is missing a windoor here
image

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by S34NW — APPROVED** [review]

**Pull request merged**